.github/workflows/build-test.yml

```diff
@@ -0,0 +1,63 @@
+name: RedBPF build test
+
+on:
+  push:
+    branches:
+      - main
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
+
+env:
+  BASE_IMAGE: ghcr.io/${{ github.repository_owner }}/ingraind-build
+
+jobs:
+  ubuntu-2004-build-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: System info
+      run: |
+        uname -a
+        lsb_release -a
+    - name: Initialize git submodules
+      run: |
+        git submodule update --init --recursive
+    - name: Run RedBPF build on ubuntu-20.04 container
+      run: |
+        docker run --privileged \
+        -v /sys/kernel/debug:/sys/kernel/debug:rw \
+        -v /lib/modules:/lib/modules:ro \
+        -v /usr/src:/usr/src:ro \
```

Comment:
Do we want to override the containers' kernels with whatever we get from GitHub? I feel like this can potentially make it harder to build for a larger range of kernels.

Going forward, I think it might be helpful to think about building for many different kernel versions to see what we can/cannot support.

Just a suggestion, though, I think it's not a straightforward thing so will be a process to get there.